include/nn/ui2d/Pane.h

```diff
@@ -5,22 +5,22 @@
 
 #pragma once
 
-#include "sead/runtime.h"  //FIXME this definitely is wrong
-#include "types.h"
+#include <nn/font/font_Util.h>
+#include <nn/types.h>
 
-namespace nn {
-namespace ui2d {
+namespace nn::ui2d {
 class AnimTransform;
 class Layout;
 
 class Pane {
+    NN_RUNTIME_TYPEINFO_BASE();
```

Comment:
the functions need to be public, imo this macro call should be moved below the `public:`